migrations/v6-MIGRATION.md

```diff
@@ -30,20 +40,19 @@
 
 ### StoreProduct changes
 
-`StoreProduct` has been made an interface, which `GoogleStoreProduct` and `AmazonStoreProduct` implement. 
-`StoreProduct` is no longer `Parcelable`.
+`StoreProduct` has been made an interface, which `GoogleStoreProduct` and `AmazonStoreProduct` implement.
 
 | Previous                      | New                                                                           |
 |-------------------------------|-------------------------------------------------------------------------------|
 | sku                           | id                                                                            |
 | price                         | Moved to `price`                                                              |
 | priceAmountMicros             | Moved to `price`                                                              |
 | priceCurrencyCode             | Moved to `price`                                                              |
-| freeTrialPeriod               | Moved to PricingPhase                                                         |
-| introductoryPrice             | Moved to PricingPhase                                                         |
-| introductoryPriceAmountMicros | Moved to PricingPhase                                                         |
-| introductoryPricePeriod       | Moved to PricingPhase                                                         |
-| introductoryPriceCycles       | Moved to PricingPhase                                                         |
+| freeTrialPeriod               | Moved to `PricingPhase` in a `SubscriptionOption`                             |
```

Comment:
i wonder if we should be more descriptive here.. we don't really have the field `freeTrialPeriod` in `PricingPhase`.

maybe we should say something like: "replaced with related PricingPhase fields". and we also have the extension `SubscriptionOptions.getFreeTrial` that we could maybe call out here